ffi/examples/read-table/read_table.c

```diff
@@ -167,6 +168,199 @@ void free_partition_list(PartitionList* list) {
   free(list);
 }
 
+
+// Our schema code might add (is nullable: x) to names, which we need to strip off.
```

Comment:
> // Our schema code might add (is nullable: x) to names, which we need to strip off.

could you give an example? I think it's `name (is nullable: true)` --> `name`?

Reply:
yeah exactly, so if the name comes in as `colName (is nullable: true)` we need to strip off the `(is nullable: true)` part to get the actual name.

Reply:
note i've now removed all this nonsense, it wasn't needed